### PR TITLE
VTCCA-3653: remove usage of SessionKeys.userId

### PR DESCRIPTION
**Pull request by PeterPerhac**

### CODE REVIEW

**Comment by habeeb01** [comment]
LGTM 👍

**Comment by darrell-plant** [comment]
Did you forget to update `navigation.scala.html` ?

File: app/actions/requests/CcaWrappedRequest.scala [hunk 1]
```diff
@@ -21,5 +21,5 @@ import models.Accounts
 trait CcaWrappedRequest {
   def optAccounts: Option[Accounts] // must be defined in implementing class
   def isLoggedIn: Boolean = optAccounts.isDefined
-  def isAgent: Boolean = optAccounts.map(_.organisation.isAgent).getOrElse(false)
+  def isAgent: Boolean = optAccounts.exists(_.organisation.isAgent)
```
Comment (darrell-plant) [inline app/actions/requests/CcaWrappedRequest.scala:24]: 👍

**Comment by darrell-plant** [comment]
LGTDz 👍

**Pull request merged**